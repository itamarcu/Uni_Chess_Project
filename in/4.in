game_mode 2
start
move <2,E> to <4,E>
move <7,D> to <6,D>
move <1,F> to <4,D>
move <1,F> to <4,C>
move <8,B> to <7,D>
move <1,G> to <3,F>
move <7,G> to <6,G>
move <3,F> to <5,G>
move <7,A> to <6,A>
move <4,C> to <7,F>
move <7,B> to <5,B>
move <7,F> to <8,E>
quit
