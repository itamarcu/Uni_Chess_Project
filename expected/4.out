 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Game mode is set to 2-player
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M M _ M M M M |
6| _ _ _ M _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Illegal move
Enter your move (white player):
8| R N B Q K B N R |
7| M M M _ M M M M |
6| _ _ _ M _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ b _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k _ n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R _ B Q K B N R |
7| M M M N M M M M |
6| _ _ _ M _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ b _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k _ n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R _ B Q K B N R |
7| M M M N M M M M |
6| _ _ _ M _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ b _ m _ _ _ |
3| _ _ _ _ _ n _ _ |
2| m m m m _ m m m |
1| r n b q k _ _ r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R _ B Q K B N R |
7| M M M N M M _ M |
6| _ _ _ M _ _ M _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ b _ m _ _ _ |
3| _ _ _ _ _ n _ _ |
2| m m m m _ m m m |
1| r n b q k _ _ r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R _ B Q K B N R |
7| M M M N M M _ M |
6| _ _ _ M _ _ M _ |
5| _ _ _ _ _ _ n _ |
4| _ _ b _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k _ _ r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R _ B Q K B N R |
7| _ M M N M M _ M |
6| M _ _ M _ _ M _ |
5| _ _ _ _ _ _ n _ |
4| _ _ b _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k _ _ r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Checkmate! white player wins the game
